RMSD.py

```diff
@@ -115,20 +115,20 @@ def RMSD(ref_mol2, query_mol2):
 
     # Split up reference and compare force fields
     refFFs = opt.ref.split(',')
-
+    
     for ref in refFFs:
         logFile.write("# Reference Force Field: %s \n" % ref)
         logFile.write("# Molecule Set Directory: %s \n" % directory)
 
         # get the list of comparison force fields and remove reference
         listFFs = opt.compare.split(',')
-        listFFs.remove(ref)
+        #listFFs.remove(ref)
```

Comment:
Oh, you probably removed it since ref won't always in the compare list, this section could be this instead:
```
if ref in listFFs: listFFs.remove(ref)
```